x/bank/keeper/msg_server.go

```diff
@@ -93,6 +124,11 @@ func (k msgServer) MultiSend(goCtx context.Context, msg *types.MsgMultiSend) (*t
 }
 
 func (k msgServer) UpdateParams(goCtx context.Context, req *types.MsgUpdateParams) (*types.MsgUpdateParamsResponse, error) {
```

Comment:
Best to switch around those two tests. So we fail first if the authority is wrong (for consistency with the other MsgUpdateParams and because it is the fastest check)